frontend/src/templates/index.mustache

```diff
@@ -11,7 +11,7 @@
     <meta name="viewport" content="width=device-width">
     <link rel="localization" href="/static/locales/{locale}/app.ftl">
 
-    <link rel="canonical" href="https://testpilot.firefox.com/">
+    <link rel="canonical" href="https://testpilot.firefox.com/{{{ canonical_path }}}">
```

Comment:
even with the trailing slash I'm not seeing the full path here, but it seems likely its just a quirk of the local server.